test/test_evaluation/test_test_evaluator.py

```diff
@@ -128,6 +128,34 @@ def test_eval_test(self):
         self.assertEqual(rval[0]["status"], StatusType.SUCCESS)
         self.assertNotIn("bac_metric", rval[0]["additional_run_info"])
 
+    def test_eval_test_multi_objective(self):
+        metrics = {
+            accuracy: 0.040000000000000036,
+            balanced_accuracy: 0.02777777777777779,
+        }
+        eval_t(
+            queue=self.queue,
+            backend=self.backend,
+            config=self.configuration,
+            metrics=list(metrics.keys()),
+            seed=1,
```

Comment:
Same here for `DEFAULT_SEED` if you agree with it.

Reply:
Done.